pkg/api/v1alpha1/fluxconfig.go

```diff
@@ -67,9 +67,6 @@ func validateFluxConfig(config *FluxConfig) error {
 }
 
 func validateGitProviderConfig(config GitProviderConfig) error {
-	if len(config.Username) <= 0 {
```

Comment:
username is not needed anymore?

Reply:
username is no longer needed -- any username information is contained in in the SSH URL provided by the user